### PR TITLE
fix(wizard): deprecate inline wizard

### LINKED ISSUE #3591: Deprecate inline wizard
Now the Stepper exists, the need for an inline wizard (which is undocumented and psuedo-supported) is no longer needed or recommended.

### PR DESCRIPTION
**Pull request by AlexMarcuBytex**
closes #3591

Signed-off-by: Marcu Alexandru <alexandru.marcu@bytex.ro>

## PR Checklist

Please check if your PR fulfills the following requirements:

* [ ] Tests for the changes have been added (for bug fixes / features)
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

* [ ] Bugfix
* [ ] Feature
* [ ] Code style update (formatting, local variables)
* [ ] Refactoring (no functional changes, no api changes)
* [ ] Build related changes
* [ ] CI related changes
* [ ] Documentation content changes
* [ ] clarity.design website / infrastructure changes
* [x] Other... Please describe:
A warning that inform the user that the inline wizard is deprecated will appear in the console if the component its used
## What is the current behavior?

Issue Number: #3591

## Does this PR introduce a breaking change?

* [ ] Yes
* [x] No

### CODE REVIEW

**Comment by AlexMarcuBytex** [comment]
@gnomeontherun can you help me with this? The error comes from the test: format script but i cant figure out what needs to be changed in the code. The logs are not clear enough.  i tried to run test: travis scripts locally, without my changes and these tests still cant be passed. What should i do to make everything work properly?

File: src/clr-angular/wizard/wizard.ts [hunk 1]
```diff
@@ -251,6 +251,11 @@ export class ClrWizard implements OnDestroy, AfterContentInit, DoCheck {
       this.listenForPageChanges()
     );
 
+    // if the inline wizard is being used, a warning will be displayed in the console
+    if (this.isStatic) {
+      console.warn('The inline wizard is considered deprecated and it will be removed in the version 4.0!');
```
[the remark this answers is not in the document]
Comment (AlexMarcuBytex) [reply src/clr-angular/wizard/wizard.ts]: no problem! im gonna change what s needed. i just did as @mathisscott proposed. ( the warning in the console ).

[the remark this answers is not in the document]
Comment (AlexMarcuBytex) [reply src/clr-angular/wizard/wizard.ts]: @gnomeontherun i investigated and i added comments to those pieces of code related to inline wizards. i have some questions about the tests. i found that the inline wizard was configured only be adding a css class to a wizard component. should i test if this class is still defined in the app or which would be the suitable approach.
Thank you!

**Comment by gnomeontherun** [comment]
Only one thing can we add a test in the deprecations.spec file that checks that the deprecated input is still there?

**Pull request merged**